src/objects/logo-group/logo-group.twig

```diff
@@ -0,0 +1,7 @@
+<div class="o-logo-group">
+  {% block content %}
+    {% for item in items %}
```

Comment:
Should this be in a demo? I'm not sure we know enough about the intended data structure to bake in support for an array into the template itself.